app/src/main/java/ar/com/wolox/android/training/ui/training/adapter/NewsServiceAdapter.kt

```diff
@@ -24,4 +25,22 @@ class NewsServiceAdapter @Inject constructor(private val retrofitServices: Retro
                 }
         )
     }
+
+    fun modifyNews(position: Int, body: NewsItem, listener: NewsPutServiceAdapterListener) {
```

Comment:
It's not a `position` what you're accepting, it's the `id`. It's not a `body` it's a `news`.